server/routes/common.js

```diff
@@ -7,8 +7,9 @@ const {
   productsByTopRatingHandler,
   productsByTrendingHandler,
 } = require('../controllers/common');
-const { signUpHandler, signInHandler } = require('../controllers');
+const { signUpHandler, signInHandler, signoutHandler } = require('../controllers');
 
+commonRouter.put('/signout', signoutHandler);
 commonRouter.post('/signup', signUpHandler);
```

Comment:
why you use the "PUT" method, I think the post is more appropriate.

[link](https://stackoverflow.com/questions/3521290/logout-get-or-post)